### PR TITLE
[tools] Revisit install_rootfs.sh

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates install_rootfs.sh script.
- Update default ubuntu version: use host codename
- Install python3-dev package for python API
- Fix hdf5 library symbolic link creation
- Update documentation

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: docs/howto/how-to-cross-build-runtime-for-aarch64.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Use `install_rootfs.sh` script to prepare Root File System. You should have `sud
 $ sudo ./tools/cross/install_rootfs.sh aarch64
 ```
 - supports `arm`(default) and `aarch64` architecutre for now
-- supports `bionic`(default) and `focal` release
+- supports `bionic`, `focal`, and `jammy` release
```
Comment (jyoungyun) [inline docs/howto/how-to-cross-build-runtime-for-aarch64.md:19]: Support Jammy 👍

File: tools/cross/install_rootfs.sh [hunk 2]
```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env bash
+
+source /etc/lsb-release
+
 usage()
 {
     echo "Usage: $0 [BuildArch] [LinuxCodeName] [--setproxy=IP] [--skipunmount]"
     echo "BuildArch can be: arm(default), aarch64"
-    echo "LinuxCodeName - optional, Code name for Linux, can be: bionic, focal(default), jammy"
+    echo "LinuxCodeName - optional, Code name for Linux, can be: bionic, focal, jammy"
```
Comment (jyoungyun) [inline tools/cross/install_rootfs.sh:9]: Now there is no default OS version in this script. OS version follows the host system. :)

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**